src/video/cameradevice.cpp

```diff
@@ -132,11 +133,20 @@ CameraDevice* CameraDevice::open(QString devName, VideoMode mode)
         }
         else
         {
-            screen = QApplication::desktop()->screenGeometry().size();
+            QScreen* defaultScreen = QApplication::screens().at(0);
```

Comment:
Will this also work in a multi-monitor environment?
Shouldn't https://doc.qt.io/qt-5/qguiapplication.html#primaryScreen-prop be used?